analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/NullPointerDereferenceTest.cs

```diff
@@ -29,23 +29,23 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class NullPointerDereferenceTest
     {
-        private readonly VerifierBuilder sonar = new VerifierBuilder()
+        private readonly VerifierBuilder sonarVerifier = new VerifierBuilder()
```

Comment:
The `Verifier` suffix is useless here

Reply:
..same goes for other rules

Reply:
`sonar` can mean anything. Everything in our solution is related to `sonar`. I think we need a better name.

Reply:
So does `Sonar` directory, this `sonar` variable is strongly bound with `Sonar` SE from `Sonar` directory and namespace, as well as `Sonar` directory for test cases.

In the same file, there's also `Roslyn` with the same binding. That could, in theory, also mean anything. But it means very specific thing in Symbolic Execution context. And it's fine to have it without that suffix. 

The thing we need to distinguish is `sonar` vs `roslyn`. Because there's nothing twice, we don't need to distinguish the type of variable (like `Verifier` vs `Rule` vs `Helper` or other duplications).

Reply:
From my point of view `sonar.Verify()` is misleading, or not telling much.

We could have 2 test classes, each with own verifier. We can have a call to brainstorm and consider other solutions as well.

Reply:
I think this really got out of hand, I hope you don't mind and I will merge the PR as it removes the OldVerifier usage, and than there are multiple things to discuss about the formatting and naming in our UTs and we can start a discussion about it. Any volunteers to drive the topic? @costin-zaharia-sonarsource  @pavel-mikula-sonarsource